hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/factories/MonkeyConstants.java

```diff
@@ -45,6 +45,11 @@ public interface MonkeyConstants {
   String UNBALANCE_WAIT_AFTER_BALANCE_MS = "unbalance.action.wait.after.period";
   String UNBALANCE_KILL_META_RS = "unbalance.action.kill.meta.rs";
   String DECREASE_HFILE_SIZE_SLEEP_TIME = "decrease.hfile.size.sleep.time";
+  String RESTART_RANDOM_RS_EXCEPTION_SLEEP_TIME = "restart.random.rs.exception.sleep.time";
```

Comment:
Can you give these configs a unit? It's always seconds? At least use `TimeUnit` to define the default values so that developer intent is clear.

Reply:
These are in milliseconds. Updated the PR to use TmeUnit.